Reason: int is 32-bits on 32-bits arches, which is not large enough to fit a
terrabyte in.
diff --git a/src/go/plugins/proc/procfs_linux.go b/src/go/plugins/proc/procfs_linux.go
index 113d395..0e8ba30 100644
--- a/src/go/plugins/proc/procfs_linux.go
+++ b/src/go/plugins/proc/procfs_linux.go
@@ -231,12 +231,13 @@ func byteFromProcFileData(data []byte, valueName string) (float64, bool, error)
 		if len(line) < 3 {
 			continue
 		}
-
-		v, err := strconv.Atoi(strings.TrimSpace(line[:len(line)-2]))
+		val, err := strconv.Atoi(strings.TrimSpace(line[:len(line)-2]))
 		if err != nil {
 			return 0, false, err
 		}
 
+		v :=  int64(val)
+
 		switch line[len(line)-2:] {
 		case "kB":
 			v *= kB
